### PR TITLE
Make options page more responsive, improve building

### PR DESCRIPTION
**Pull request by yayuyokitano**
Implements a bottom context menu in place of sidebar on low widths for better experience. Still not 100% ideal but pretty close.

Example:
image

Also adds a dev mode to safari build.

Also adds new build target as discussed.

### CODE REVIEW

File: build.ts [hunk 1]
```diff
@@ -40,20 +40,32 @@ async function main() {
 	];
 	await Promise.all(scripts);
 
+	if (isDev() && releaseTarget === 'safari') {
+		await new Promise((resolve) => {
```
Comment (inverse) [inline build.ts:44]: Why do we need to have this sleep?

**Review by inverse — APPROVED** [review]
LGTM - nice one :+1:

File: build.ts (hunk 1 above)
Comment (yayuyokitano) [reply build.ts:44]: The dev build does not await the completion of the build before continuing, the sleep allows the initial build processes to finish up before creating the xcode project.

There is probably a better way here long term, but this did nicely for development on my end for the time being.

Comment (inverse) [reply build.ts:44]: Ahhh is that because of the watch mode? Yeah we should probably think of a better way longer term - but right now it'll do :)

**Pull request merged**